include/avif/avif.h

```diff
@@ -526,7 +526,10 @@ typedef struct avifReformatState
     int rgbMaxChannel;
     float yuvMaxChannelF;
```

Comment:
Could you find out if we still need the yuvMaxChannelF field in this struct?

Reply:
It is used as an intermediate value to initialize `yRange`, `uvRange`, `unormFloatTableY` and `unormFloatTableUV`. Maybe we can make it a local variable of `avifPrepareReformatState`.

Reply:
After update only initialization of `yRange` and `uvRange` uses `yuvMaxChannelF`. But maybe I'll left it here for completeness. By the way, `rgbDepth` is not used anywhere.